### DISCUSSION TITLE
Is there a reason for choosing turborepo instead of nx?

### DISCUSSION THREAD

**Discussion by LoyalPotato** [nandorojo/solito#96] [Q&A] [answered]
What it says in the title.

I was asking to know if you/anyone had any experience setting up the same environment with nx and if they had any issues with it or not.

**Comment by nandorojo** [comment, marked as answer]
i haven’t used nx. i like turbo. really simple and works well with vercel

**Reply by LoyalPotato** [reply]
So it was just a simple choice, not really comparing the other solutions?

**Reply by nandorojo** [reply]
yeah. turbo’s presence in the repo is very minimal, so it should be easy to change if you want to

**Reply by LoyalPotato** [reply]
I see. Thanks for the quick replies :)
